### PR TITLE
Frishs solution

### PR DESCRIPTION
**Pull request by LaFrish**
completeness 4
comfort level 4

### CODE REVIEW

**Comment by LaFrish** [comment]
Work was done in the obfuscated file

File: obfuscate/atm.js [hunk 1]
```diff
@@ -1 +1,37 @@
...
+  savingTotal += parseFloat(savingUserInput.val());
+  savingDisplay.html(savingTotal)
+  console.log(savingUserInput.val());
+})
+
+savingWithdrawlButton.on("click", function(){
+  savingTotal -= parseFloat(savingUserInput.val());
+  savingDisplay.html(savingTotal)
+  console.log(savingUserInput.val())
+})
```
Comment (jshawl) [inline obfuscate/atm.js:37]: holy smokes this is some clean code!!!

File: obfuscate/atm.js [hunk 2]
```diff
@@ -1 +1,37 @@
...
+var checkingTotal = 0
+var checkingDisplay = $("#checking_balance")
+
+var savingDepositButton = $("#savings_deposit")
+var savingWithdrawlButton = $("#savings_withdraw")
+var savingUserInput = $("#savings_amount")
+var savingTotal = 0
+var savingDisplay = $("#savings_balance")
+
+var negative = $(".zero")
```
Comment (jshawl) [inline obfuscate/atm.js:13]: is this being used anywhere?

File: obfuscate/atm.js [hunk 3]
```diff
@@ -1 +1,37 @@
...
+var checkingWithdrawlButton = $("#checking_withdraw")
+var checkingUserInput = $("#checking_amount")
+var checkingTotal = 0
+var checkingDisplay = $("#checking_balance")
+
+var savingDepositButton = $("#savings_deposit")
+var savingWithdrawlButton = $("#savings_withdraw")
+var savingUserInput = $("#savings_amount")
+var savingTotal = 0
+var savingDisplay = $("#savings_balance")
```
Comment (jshawl) [inline obfuscate/atm.js:11]: defining all your vars up top makes the code super easy to read. If you wanted to get fancy, you could combine these variables into an object, e.g:

``` js
var accounts = {
  checking: {
    depositButton: ,
    balance: 0
  } 
  savings: ... etc
}
```

**Comment by jshawl** [comment]
:star2: excellent job on this assignment!!

**Pull request closed without merging**